File: examples-sdk/csharp/703-protocol-soap-client/Main.cs
// 703-protocol-soap-client — C# SDK equivalent
// Compile: vil compile --from csharp --input 703-protocol-soap-client/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();

File: examples-sdk/java/703-protocol-soap-client/Main.java
// 703-protocol-soap-client — Java SDK equivalent
// Compile: vil compile --from java --input 703-protocol-soap-client/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}
